models/evc.py

```diff
@@ -650,15 +650,29 @@ def remove_path_flows(self, path=None, force=True):
             return
 
         dpid_flows_match = {}
-        for dpid, flows in self._prepare_nni_flows(path).items():
+
+        try:
+            nni_flows = self._prepare_nni_flows(path)
+        except Exception as err:
```

Comment:
@italovalcy, the intention is clear that you'd like to catch any error, including unexpected ones just so it doesn't impact in this operation, which is OK with this goal in mind, but notice that catching `Exception` for unexpected error it can make it difficult to trace certain unexpected errors, for instance let's say this error here where I accidentally introduced an unused var (and let's say unit test didn't catch it either):

```
diff --git a/models/evc.py b/models/evc.py
index a72043f..5b48623 100644
--- a/models/evc.py
+++ b/models/evc.py
@@ -653,6 +653,7 @@ class EVCDeploy(EVCBase):
 
         try:
             nni_flows = self._prepare_nni_flows(path)
+            a
         # pylint: disable=broad-except
         except Exception as err:
             log.error(f"Fail to remove NNI failover flows for {self}: {err}")
```

Since it's an undefined var we'd be able to grep and eventually find it, but you can imagine that depending on the underlying exception that the context might not be much helpful, I'll recommend that you also include the traceback in the logs too just so we have sufficient information to fix any potential unexpected issue, we've used this approach on [`listen_to` handler context](https://github.com/kytos-ng/kytos/blob/master/kytos/core/helpers.py#L146-L148), wdyt?

```
2023-09-26 17:00:34,683 - ERROR [kytos.napps.kytos/mef_eline] (thread_pool_app_11) Fail to remove NNI failover flows for EVC(e867d3057dbf42, evpl): name 'a' is not defined
```

Reply:
Excellent suggestion, @viniarck. Thanks for the detailed explanation and provided example. I've submitted commit 7d182cb to fix this issue. Please let me know if you have concerns with that approach.

Reply:
Looks good. It's easier to trace potential issues now:

```
2023-09-27 10:27:15,518 - ERROR [kytos.napps.kytos/mef_eline] (thread_pool_app_0) Fail to remove NNI failover flows for EVC(77e2d4edaaf54f, evpl): Traceback (most recent call last):,   F
ile "/home/viniarck/repos/napps/napps/kytos/mef_eline/models/evc.py", line 657, in remove_path_flows,     a, NameError: name 'a' is not defined, 
```